devicetypes/smartthings/hue-bulb.src/hue-bulb.groovy

```diff
@@ -23,8 +23,8 @@ metadata {
 		// TODO: define status and reply messages here
 	}
 
-	tiles (scale: 2){
-		multiAttributeTile(name:"switch", type: "lighting", width: 6, height: 4, canChangeIcon: true){
+	tiles(scale: 2) {
+		multiAttributeTile(name:"rich-control", type:"lighting", canChangeIcon: true) {
```

Comment:
We specifically had the name stay as "switch" to maintain compatibility with older versions. It probably doesn't matter now since we added a fix to the platform to handle this potential error case.